.travis.yml

```diff
@@ -19,5 +22,14 @@ before_cache:
   - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
   - find $HOME/.sbt -name "*.lock" -delete
 
+# For a PR TRAVIS_BRANCH is set to the target branch which is master.
+# We need to make sure it's not a PR when releasing.
 script:
-   - sbt ++$TRAVIS_SCALA_VERSION ciBuild
+  - if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ];
+    then
+      docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
+      sbt ++$TRAVIS_SCALA_VERSION ciRelease;
```

Comment:
You're changing the behaviour to do a release for every single push to master, which conflicts with the notes in `CONTRIBUTING.md`.  I'm open minded to discussing moving in this direction, but it needs to be a conscious decision (and updated consistently across the board) rather than just a side-effect of adding Docker publishing logic 😃